datumaro/plugins/splitter.py

```diff
@@ -13,34 +14,172 @@
 
 NEAR_ZERO = 1e-7
 
+SplitTask = Enum(
+    "split", ["classification", "detection", "segmentation", "reidentification"]
+)
 
-class _TaskSpecificSplit(Transform, CliPlugin):
-    _default_split = [('train', 0.5), ('val', 0.2), ('test', 0.3)]
+
+class Split(Transform, CliPlugin):
+    """
```

Comment:
I think this can be too big output for a CLI help, but let's keep it this way for now. However, align it to 80 characters per line.

Reply:
https://github.com/openvinotoolkit/datumaro/pull/223/commits/1d45ec6ba1b79a22333ec5713bff3671d69ed349
aligned it.